packages/shared/lib/core/wallet/actions/sendAmount.ts

```diff
@@ -0,0 +1,18 @@
+import { selectedAccount } from '@core/account'
+import { AddressWithAmount, TransactionReceipt, TransferOptions } from '@iota/wallet'
+import { get } from 'svelte/store'
+
+export function sendAmount(receipientAddress: string, amount: number): Promise<TransactionReceipt[]> {
+    const account = get(selectedAccount)
+    const addressWithAmount: AddressWithAmount = { address: receipientAddress, amount: amount.toString() }
+    const transferOptions: TransferOptions = {
+        remainderValueStrategy: { strategy: 'ReuseAddress', value: null },
```

Comment:
Getting an error here:
```
ERROR in /home/maxwellmattryan/dev/iota/repos/firefly-pr/packages/shared/lib/core/wallet/actions/sendAmount.ts
[tsl] ERROR in /home/maxwellmattryan/dev/iota/repos/firefly-pr/packages/shared/lib/core/wallet/actions/sendAmount.ts(9,35)
      TS2322: Type '{ strategy: string; value: null; }' is not assignable to type 'RemainderValueStrategy'.
  Object literal may only specify known properties, and 'strategy' does not exist in type 'RemainderValueStrategy'.
```

Reply:
The typing in the dependency is wrong waiting for David to update it.